src/pages/workspace/accounting/qbo/import/QuickbooksTaxesPage.tsx

```diff
@@ -42,12 +45,14 @@ function QuickbooksTaxesPage({policy}: WithPolicyProps) {
                             <View style={[styles.flex1, styles.alignItemsEnd, styles.pl3]}>
                                 <Switch
                                     accessibilityLabel={translate('workspace.accounting.taxes')}
-                                    isOn={!!syncTax}
+                                    isOn={isTaxSwitchOn}
```

Comment:
should we manually make the toggle go off on the front end side when the Journal Entry is selected?

Reply:
Reading @hayata-suenaga [comment](https://github.com/Expensify/App/pull/41520#issuecomment-2093294494), and looking here... I think we should do this. If the `syncTax` is not configured correctly, we will hide it from the user and potentially have errors in the backend.

I would prefer to see that the setting is wrong in the front end so the problem is more visible and we can fix it.

Reply:
> should we manually make the toggle go off on the front end side when the Journal Entry is selected? 

If enabled, why don’t we `disable` the `syncTax` when we update the policy connection configuration to set `Journal entry` ? This will make the toggle go off in FE and align the BE correctly with the `tax import` configuration. This way we also do not have to worry about the incorrect configuration of `tax import` in BE and the related complication as mentioned [here](https://github.com/Expensify/App/pull/41520#discussion_r1589746248). 
Does this sound reasonable? 
cc @hayata-suenaga @aldo-expensify @trjExpensify 

If we decide to go this way, we may also want to display a warning message that `syncTax` will be reset if we select `Journal entry`.

Reply:
> If enabled, why don’t we disable the syncTax when we update the policy connection configuration to set Journal entry ?

Do you mean making this change in the backend 😄? We have different APIs for Expensify Classic and New Expensify, so this still doesn't prevent the misconfigurations.

> If we decide to go this way, we may also want to display a warning message that syncTax will be reset if we select Journal entry.

We already prevent the users from selecting Journal Entry when tax sync (i.e. import) is enabled. What we're tying to address here are edges cases. 😄